src/SonarQube.TeamBuild.PostProcessor/SonarQube.TeamBuild.PostProcessor.csproj

```diff
@@ -9,4 +9,17 @@
     <ProjectReference Include="..\SonarQube.TeamBuild.Integration\SonarQube.TeamBuild.Integration.csproj" />
     <ProjectReference Include="..\SonarScanner.Shim\SonarScanner.Shim.csproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
VS keeps adding it every time a resx file changes, but it's not necessary.
We can keep removing it (we don't seem to edit resx too often, so it is ok), or add this bit of xml to every csproj that has resources and remove the bit of magic from `Directory.Build.targets`. I'd go for first option, WDYT?